api/app.js

```diff
@@ -21,11 +21,12 @@ app.use(cors())
 
 // Start app on environment port
 app.listen(config.PORT, async function () {
-    console.log(`Listening on port ${config.port}`);
-    console.log(`Connecting to mongo`);
+    console.log(`Listening on port ${config.PORT}`);
+    console.log(`Connecting to mongo...`);
     const { MONGO_PORT, MONGO_DB_NAME, MONGO_HOST } = config
     try {
-        await mongoose.connect(`mongodb://${MONGO_HOST}:${MONGO_PORT}/${MONGO_DB_NAME}`);
+        // await mongoose.connect(`mongodb://${MONGO_HOST}:${MONGO_PORT}/${MONGO_DB_NAME}`);
```

Comment:
Don't do it like this. Change the mongo host, port and dbname in the config.json
Also remove the line you added and restore to previous